packages/api/src/external/carequality/command/cq-directory/search-cq-directory.ts

```diff
@@ -25,24 +25,30 @@ export type CQOrgBasicDetails = {
  * @param cxId The ID of the customer organization.
  * @param patientId The ID of the patient.
  * @param radiusInMiles Optional, the radius in miles within which to search for organizations. Defaults to 50 miles.
+ * @param updatePatientAddresses Optional, whether to update the patient's addresses with coordinates if they don't already exist. Defaults to false.
+ * @param reportRelevance Optional, whether to report low relevance addresses to the cx. Defaults to false.
  *
  * @returns Returns the details of organizations within the specified radius of the patient's addresses.
  */
-export async function searchNearbyCQOrganizations({
+export async function addPatientCoordinatesAndSearchNearbyCQOrganizations({
   cxId,
   patientId,
   radiusInMiles = DEFAULT_RADIUS_IN_MILES,
+  updatePatientAddresses = false,
+  reportRelevance = false,
```

Comment:
nit: I feel bad about this, but now that we're making updating the patient's address something optional (and defaults not to update), starting the function name with the operation that by default is not executed is odd.

Trying to think more abstractly what this does, I checked where this is used, and nowhere we ask it to update the patient's address. So maybe this should be removed to simplify the function?

Reply:
Yea absolutely